services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/container.py

```diff
@@ -0,0 +1,97 @@
+from typing import Any, Dict, Union
+
+import aiodocker
+from fastapi import APIRouter, Query, Request, Response
+
+from ..storage import SharedStore
+
+container_router = APIRouter()
+
+
+@container_router.get("/container/logs")
+async def get_container_logs(
+    # pylint: disable=unused-argument
+    request: Request,
+    response: Response,
+    container: str,
+    since: int = Query(
```

Comment:
what is /container for?
in the case of multiple containers what is this one?

would it not make more sense to /containers/{name or id}/logs ...?

Reply:
you are correct about this. will change it to "/container/{name_or_id}/logs" and all subsequent entries in this module.